src/main/java/net/datafaker/providers/videogame/Dota2.java

```diff
@@ -0,0 +1,52 @@
+package net.datafaker.providers.videogame;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+ * Dota 2 is a multiplayer online battle arena (MOBA) game developed and published by Valve Corporation.
+ *
+ * @author panic08
+ * @since 2.3.1
+ */
+public class Dota2 extends AbstractProvider<VideoGameProviders> {
+
+    protected Dota2(VideoGameProviders faker) {
+        super(faker);
+    }
+
+    public String fraction() {
+        return resolve("games.dota2.fraction");
+    }
```

Comment:
Do you mean "faction" ?

Reply:
Yes, fraction/faction

https://dota2.fandom.com/wiki/Factions

Reply:
Are they called "fractions" in the game (with an R)?

Because I didn't see it on that Fandom page.

I'm happy to leave it if it's topical, I just don't know. I'm not a DOTA player.

Reply:
Yes, you're right. I've now manually checked it myself and indeed “faction” is correct